Tools/PostProcessing/plot_distribution_mapping.py

```diff
@@ -69,19 +69,22 @@ def _get_costs_reduced_diagnostics(self, directory, prange):
             unique_headers=[''.join([l for l in w if not l.isdigit()])
                             for w in h.split()][2::]
 
-        # Either 7 or 8 depending if GPU
-        n_data_fields = 7 if (len(set(unique_headers)) - 2)%7 == 0 else 8
```

Comment:
Since we remove the first two columns already two lines above, this is likely wrong already in the original code and should not have `- 2`

Reply:
fixed in this PR